### DISCUSSION TITLE
Unikernel deployed on K8s

### DISCUSSION THREAD

**Discussion by chenzq-98** [unikraft/unikraft#654] [Q&A]
I see the runu for unikernel in https://www.youtube.com/watch?v=cV-xawN9_cg&t=33s. Is the runu tool available for us?

**Comment by razvand** [comment]
@czq693497091, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
